Classes/WPEditorFormatbarView.m

```diff
@@ -350,12 +359,12 @@ - (void)initBarButtonItem:(ZSSBarButtonItem*)barButtonItem
     customButton.normalTintColor = self.itemTintColor;
     customButton.selectedTintColor = self.selectedItemTintColor;
     customButton.disabledTintColor = self.disabledItemTintColor;
-    customButton.imageView.contentMode = UIViewContentModeCenter;
+    customButton.imageView.contentMode = UIViewContentModeScaleAspectFit;
     [customButton addTarget:target
                      action:selector
            forControlEvents:UIControlEventTouchUpInside];
     barButtonItem.customView = customButton;
-    barButtonItem.width = buttonImage.size.width;
+    barButtonItem.image = image;
```

Comment:
How come we're setting the image on the bar button item? Isn't that not displayed if we're using a `customView`?